### PR TITLE
add missing closed bracket

### PR DESCRIPTION
**Pull request by bytesun**
it cause the link can't be open (404 error)

### CODE REVIEW

File: doc/modules/language-guide/pages/basic-concepts.adoc [hunk 1]
```diff
@@ -44,7 +44,7 @@ In preparing for that discussion, we discuss programs in this chapter and in lin
 Rather, these tiny programs illustrate snippets of {proglang} for writing those services, and each can (usually) be run on its own as a (non-service) {proglang} program, possibly with some printed terminal output.
 
 The examples in this section illustrate basic principles using simple expressions, such as arithmetic.
-For an overview of the full expression syntax of {proglang}, see the link:langauge-manual{outfilesuffix[Lanaguage quick reference].
+For an overview of the full expression syntax of {proglang}, see the link:langauge-manual{outfilesuffix}[Lanaguage quick reference].
```
Comment (nomeata) [inline doc/modules/language-guide/pages/basic-concepts.adoc:47]:
```suggestion
For an overview of the full expression syntax of {proglang}, see the link:langauge-manual{outfilesuffix}[Language quick reference].
```

**Review by ggreif — APPROVED** [review]

File: doc/modules/language-guide/pages/basic-concepts.adoc [hunk 2]
```diff
@@ -44,7 +44,7 @@ In preparing for that discussion, we discuss programs in this chapter and in lin
 Rather, these tiny programs illustrate snippets of {proglang} for writing those services, and each can (usually) be run on its own as a (non-service) {proglang} program, possibly with some printed terminal output.
 
 The examples in this section illustrate basic principles using simple expressions, such as arithmetic.
-For an overview of the full expression syntax of {proglang}, see the link:langauge-manual{outfilesuffix[Lanaguage quick reference].
+For an overview of the full expression syntax of {proglang}, see the link:langauge-manual{outfilesuffix}[Language quick reference].
```
Comment (ggreif) [inline doc/modules/language-guide/pages/basic-concepts.adoc:47]:
```suggestion
For an overview of the full expression syntax of {proglang}, see the link:language-manual{outfilesuffix}[Language quick reference].
```

**Comment by nomeata** [comment]
I changed something about the nix cache (moved it into its own “organization“ on cachix so that it doesn't count towards my personal storage allowance), and something broke along the way. I notified the cachix people.

**Comment by ggreif** [comment]
Thanks @bytesun!

**Pull request merged**